test/workflow_test.go

```diff
@@ -966,6 +966,51 @@ func (w *Workflows) WorkflowWithLocalActivityRetries(ctx workflow.Context) error
 	return nil
 }
 
+func (w *Workflows) WorkflowWithLocalActivityRetriesAndDefaultRetryPolicy(ctx workflow.Context) error {
+	laOpts := w.defaultLocalActivityOptions()
+	// Don't set any retry policy
+	ctx = workflow.WithLocalActivityOptions(ctx, laOpts)
```

Comment:
Could you add a unit test using TestWorkflowEnvironment just to verify unit test works as well.

Reply:
Yes, will do.